tpm2_pytss/ESAPI.py

```diff
@@ -2431,21 +2431,31 @@ def PP_Commands(
         session3=ESYS_TR.NONE,
     ):
 
+        check_handle_type(auth, "auth")
+        setList_cdata = get_cdata(setList, TPML_CC, "setList")
+        clearList_cdata = get_cdata(clearList, TPML_CC, "clearList")
         _chkrc(
             lib.Esys_PP_Commands(
-                self.ctx, auth, session1, session2, session3, setList, clearList
+                self.ctx,
+                auth,
+                session1,
```

Comment:
Why not default session1 to password session?

Reply:
PP_Commands seems to require a policy session as you have to assert physical presence

Reply:
Duh, I misread the test, that's a negative test:
```python
        with self.assertRaises(TSS2_Exception) as e:
```